src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,48 @@ However, we have retracted this method for the following reasons:
 * Various studies, such as [Baker et al.](https://www.acpjournals.org/doi/10.7326/M20-7145) have shown that, for at least some pockets of healthcare workers, healthcare workers are no more likely to get infected through their work than the average person is to get infected through socializing.
 * The above Washington State report shows that people who work outside the home got COVID at about the average rate for the state.
 
+# Vaccines Q&A
+
+## How should we think about people who have been vaccinated?
+
+**TL;DR**: Vaccinated people receive a multiplier on all incoming microCOVIDs from their activities. The multiplier depends on which vaccine they've gotten and is contingent on having waited long enough for their immune system to respond. Additionally, each microCOVID *may* be 2-4x less likely to land a vaccinated individual in a hospital.
```

Comment:
Could we call this "summary" so it's less techy-sounding? And could we italicize it so it's clear where the summary ends and the main section starts?